src/middlewared/middlewared/plugins/vm/lifecycle.py

```diff
@@ -64,6 +64,8 @@ def initialize_vms(self, timeout=30):
                     self.middleware.logger.error(
                         'Unable to setup %r VM object: %s', vm_data['name'], str(e), exc_info=True
                     )
+            self.middleware.call_sync('service.reload', 'http')
```

Comment:
What's the point of adding haproxy if we keep reloading nginx?

Reply:
We were only doing it once ideally in the middleware lifecycle. I can remove this and always have the vm endpoint in nginx.